ibm_cloud_sdk_core/utils.py

```diff
@@ -50,3 +53,109 @@ def string_to_datetime(string):
     :return: datetime.
     """
     return date_parser.parse(string)
+
+def get_authenticator_from_environment(service_name):
+    """
+    Checks the credentials file and VCAP_SERVICES environment variable
+    :param service_name: The service name
+    :return: the authenticator
+    """
+    authenticator = None
+    # 1. Credentials from credential file
+    config = read_from_external_sources(service_name)
+    if config:
+        authenticator = contruct_authenticator(config)
+
```

Comment:
After the credentials file, we should be looking at generic environment variables